packages/esm-patient-list-app/src/PatientTable/patient-table.component.tsx

```diff
@@ -0,0 +1,140 @@
+import { navigate } from '@openmrs/esm-framework';
+import { Link } from 'carbon-components-react';
+import DataTable, {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from 'carbon-components-react/lib/components/DataTable';
+import DataTableSkeleton from 'carbon-components-react/lib/components/DataTableSkeleton';
+import Pagination from 'carbon-components-react/lib/components/Pagination';
+import Search from 'carbon-components-react/lib/components/Search';
+import { debounce } from 'lodash';
+import React, { useMemo, CSSProperties } from 'react';
+import styles from './patient-table.component.scss';
+
+const PatientTable: React.FC<PatientTableProps> = ({ patients, columns, search, pagination, isLoading, autoFocus }) => {
+  const rows: Array<any> = useMemo(
+    () =>
+      patients.map((patient) => {
+        const row = {};
+        columns.forEach((column) => {
+          row['id'] = patient['id'] || patient['uuid'];
```

Comment:
This does not have to be done for every column.

Reply:
Nice catch!